icu4c/source/allinone/Build.Windows.PlatformToolset.props

```diff
@@ -18,9 +18,4 @@
   <PropertyGroup Label="PlatformToolset">
     <PlatformToolset>$(DefaultPlatformToolset)</PlatformToolset>
   </PropertyGroup>
-  <PropertyGroup>
-    <!-- Disable MSBuild warning about Linker OutputFile. -->
-    <!-- Ex: MSBuild complains that the common project creates "icuuc62.dll" rather than "common.dll". However, this is intentional. -->
-    <MSBuildWarningsAsMessages>MSB8012</MSBuildWarningsAsMessages>
-  </PropertyGroup>
```

Comment:
I think deleting the warning suppression in this file brought back some of the MSB8012 warnings.

Reply:
Oh good point. I moved it into the "library" file, but not all the projects include the "library" props file. I think we'll need to keep it in both files for now.

Thanks for pointing this out!

Reply:
Please take a look at PR #373. 
(Thanks again!)

Reply:
Thanks for making the fix! I approved PR #373. :)